routes/web.php

```diff
@@ -19,6 +19,7 @@
 use App\Http\Controllers\StatuslabelsController;
 use App\Http\Controllers\SuppliersController;
 use App\Http\Controllers\ViewAssetsController;
+use App\Http\Controllers\Licenses\LicenseCheckinController;
```

Comment:
If you're calling it from the API url, the controller method needs to be in the API controller namespace and `api.php route`.

Reply:
Method has been moved to the API controller and this namespace has been changed.